diff --git a/apps/micro-react/.env b/apps/micro-react/.env
index a43d6bb..8b93666 100644
--- a/apps/micro-react/.env
+++ b/apps/micro-react/.env
@@ -1 +1,2 @@
-PORT=3001
\ No newline at end of file
+PORT=3001
+REACT_APP_BASE_URL = http://localhost:3000/
diff --git a/apps/micro-react/src/common/request/index.ts b/apps/micro-react/src/common/request/index.ts
index 917d541..0ea37da 100644
--- a/apps/micro-react/src/common/request/index.ts
+++ b/apps/micro-react/src/common/request/index.ts
@@ -42,7 +42,7 @@ responseInterceptors.forEach((interceptor) => {
   interceptors.response.use(interceptor);
 });

-const baseUrl = 'http://localhost:3000/';
+const baseUrl = process.env.REACT_APP_BASE_URL;

 const abortMap = new Map();

diff --git a/apps/react-base/src/common/request/index.ts b/apps/react-base/src/common/request/index.ts
index 1988d9e..50dea5e 100644
--- a/apps/react-base/src/common/request/index.ts
+++ b/apps/react-base/src/common/request/index.ts
@@ -35,7 +35,7 @@ forEach(responseInterceptors, (interceptor) => {
   interceptors.response.use(interceptor);
 });

-const baseUrl = 'http://localhost:3000/';
+const baseUrl = process.env.REACT_APP_BASE_URL;

 const abortMap = new Map();
 const request = <T = any>(








